gtest/src/log.rs

```diff
@@ -21,8 +23,44 @@ impl CoreLog {
             destination: other.destination(),
             payload: other.payload().to_vec(),
             exit_code: other.exit_code(),
+            id: other.id(),
         }
     }
+
+    pub(crate) fn from_stored_message(other: StoredMessage) -> Self {
```

Comment:
You can implement `From<StoredMessage> for CoreLog` as a more flexible conversion way.

Reply:
Agree to replace this method and `from_message` in order to provide `From` impl